controller/launch.py

```diff
@@ -267,6 +373,30 @@ def main() -> None:
         logger.error("Invalid configuration: %s", e)
         sys.exit(1)
 
+    # Parse sleep manager configuration
+    sleep_manager_config = _parse_sleep_manager_config(raw_cfg)
+
+    # Extract model configurations for sleep manager
+    vllm_models_config, sglang_models_config = _extract_model_configs_for_sleep_manager(
+        instances_cfg)
```

Comment:
Is this overlapping with `instances_cfg = _parse_cfg(raw_cfg, cfg_path.parent)`? I think we don't need to parse the .yaml again; instead, directly get the config from `instances_cfg`?